src/lib/acronyms/data.json

```diff
@@ -0,0 +1,86 @@
+  "ACB": "Artsy Confirmed Buyer",
+  "ACV": "Annual Contract Value",
+  "Av.CV": "Average Contract Value",
+  "AH": "Auction House",
+  "AI": "Action Item",
+  "AML": "Anti-money laundering ",
+  "AMP": "Artsy Marketplace Partners",
+  "ARR": "Annual Recurring Revenue ",
+  "AOV": "Average Order Value",
+  "APAC": "Asia Pacific",
+  "BNMO": "Buy Now, Make Offer (outdated / replaced by NSO) ",
+  "BOGO": "Buy One, Get One ",
+  "B2B": "Business to Business",
+  "B2C": "Business to Customer",
+  "CAC": "Customer Acquisition Cost",
+  "CCE": "Curation, Collection Engagement (outdated -- now Curatorial) ",
+  "CLC": "Collector Live Chat (Chat for collectors on Artsy.net) ",
+  "COA": "Certificate of Authenticity",
+  "CCPA": "California Consumer Privacy Act (took effect 1/1/2020)",
+  "CPA": "Cost per Acquisition",
+  "CMS": "Content Management System (cms.artsy.net; where Gallery partners update their profile)",
+  "CRT": "Collector Relations Team",
+  "CSAT": "Customer Satisfaction Rating",
+  "CTR": "Click Through Rate",
+  "CVP": "Core Value Proposition ",
+  "CVR": "Conversion Rate",
+  "DRI": "Directly Responsible Individual ",
+  "DEI": "Diversity, Equity, and Inclusion",
+  "EBC": "Editorial, Brand, Content",
+  "EBITDA": "Earnings before interest, taxes, depreciation, and amortization",
+  "EMEA": "Europe, Middle East, Africa",
+  "ERP": "Enterprise Resource Planning",
+  "EOY/EOD": "End of year / end of day ",
+  "FWIW": "For what it’s worth",
+  "GDPR": "General Data Protection Regulation",
+  "GMV": "Gross Merchandise Value",
+  "GRC": "Gallery Resource Center",
+  "GRCD": "Gallery Relations Commission Dashboard ",
+  "GSP": "Global Strategic Partnerships (Outdated team name → has become Large Accounts, Fairs)",
+  "HQ": "Headquarter",
+  "INA": "Item Not Available",
+  "IO": "Insertion Order",
+  "IRL": "in real life ",
+  "KPI": "Key Performance Indicator ",
+  "LDOM": "Last Day of the Month",
+  "LTM": "Last Twelve Months",
+  "LTV": "Lifetime value ",
+  "MAU": "Monthly Active Users",
+  "MSO": "Marketplace Strategy and Operations ",
+  "MPC": "Monthly Partner Communication (Outdated Term; now Director’s Digest)",
+  "MQL": "Marketing Qualified Lead ",
+  "MRR": "Monthly Recurring Revenue ",
+  "MVE": "Minimal Viable Experiment",
+  "NALA": "North America, Latin America",
+  "NPS": "Net promoter score",
+  "NNTR": "No Need To Respond",
+  "NRR": "Net Revenue Retention ",
+  "NSO": "New Selling Options ",
+  "OKR": "Objectives and Key Results (how Artsy sets goals)",
+  "OTE": "On-target earnings (in the context of commission plan)",
+  "OOO": "Out of Office",
+  "PAC": "Partner Applied Categories",
+  "PDDE": "Product Design Data Engineering",
+  "PII": "Personally identifiable information (e.g. name, email, birthdate, etc.)",
+  "P&C": "People & Culture",
+  "ROI": "Return On Investment",
+  "SEO": "Search Engine Optimization",
+  "SALs": "Sales Accepted Leads",
+  "SLAs": "Service Level Agreements",
```

Comment:
We might want to remove the trailing `s` from these two acronyms. I image `SLA` and `SAL` would be the common queries.